ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Synchronize.scala

```diff
@@ -41,4 +43,20 @@ object Synchronize {
       // point before invoking this method
     }
   }
+
+  final def waitForContract[T](
+      alpha: ParticipantTestContext,
```

Comment:
Please rename to `participant` or something like that.

Reply:
done